src/profile-logic/processed-profile-versioning.js

```diff
@@ -818,5 +818,39 @@ const _upgraders = {
       thread.stringArray = stringTable.serializeToArray();
     }
   },
+  [18]: profile => {
+    // funcTable gains a new field: columnNumber.
```

Comment:
please add a comment explaining a bit more what we're doing here. Something like this:
```js
// When we added column numbers we forgot to update the func table.
// As a result, when we had a column number for an entry, the line number
// ended up in the `fileName` property, and the column number in the
// `lineNumber` property.
```